frontend/src/components/alert/snackBar.tsx

```diff
@@ -8,19 +8,22 @@ interface snackbarProps {
   message: string;
   severity: string;
   handleClose: () => void;
+  autoHideDuration?: number | null;
```

Comment:
so what happens when it's null?

Reply:
https://material-ui.com/api/snackbar/ if that is null then snackbar doesn't auto hide